cpu/ezr32wg/Makefile

```diff
@@ -4,4 +4,7 @@ MODULE = cpu
 # add a list of subdirectories, that should also be build
 DIRS = periph $(RIOTCPU)/cortexm_common
 
+#
```

Comment:
> What happens when you compile vectors.c with LTO?
> Do you know if it is enough to add a attribute used tags somewhere
> or if there's more to it?

Somehow it triggers this error:

    lto1: internal compiler error: in add_symbol_to_partition_1, at
lto/lto-partition.c:135
    Please submit a full bug report,
    with preprocessed source if appropriate.
    See <https://bugs.archlinux.org/> for instructions.
    lto-wrapper: fatal error: arm-none-eabi-gcc returned 1 exit status
    compilation terminated.
    /usr/lib/gcc/arm-none-eabi/6.1.1/../../../../arm-none-eabi/bin/ld:
error: lto-wrapper failed
    collect2: error: ld returned 1 exit status
    make: ***
[/home/kaspar/src/riot.tmp/examples/default/../../Makefile.include:261:
all] Error 1